test/ctia/stores/es/wait_for_test.clj

```diff
@@ -0,0 +1,31 @@
+(ns ctia.stores.es.wait-for-test
```

Comment:
Just one detail to discuss before you merge it.
Even if the feature is closely tied to ES, this test a feature at the route level. 
Perhaps it belongs more to `ctia.http.routes.common-test`.
wdyt?

Reply:
Yes, makes sense.